src/DarkNet/models/DarknetServerData.ts

```diff
@@ -0,0 +1,171 @@
+import { getRandomIcon, Minigames } from "../controllers/ServerGenerator";
+import { Icon, labIcon } from "../ui/ServerIcon";
+import { AddToAllServers, createUniqueRandomIp, GetServer } from "../../Server/AllServers";
+import {
+  commonPasswordDictionary,
+  connectors,
+  l33t,
+  loreNames,
+  presetNames,
+  ServerNamePrefixes,
+  ServerNameSuffixes,
+} from "./dictionaryData";
+import { getLabyrinthDetails } from "../effects/labyrinth";
+import { DarknetServer } from "../../Server/DarknetServer";
+
+export const ResponseStatus = {
```

Comment:
There isn't a DarkNet/Enums.ts, or a corresponding entry in the global Enums.ts, which I find surprising. I suspect this is because enums and enum-like things have been scattered throughout the code.

I think we *do* want them globally registered like that, though - as the comment in Enums.ts says, it ensures that we don't accidentally step on our own toes with duplicate names. But, as I said in discord I think we want "new style" enums (like what you're doing here) and not old-style ts enums. The file structure can be the same, though.

Reply:
I've moved a bunch of them to the Darknet/enums.ts